src/helpers/experimental.js

```diff
@@ -46,13 +52,27 @@ export const experimentalFeatures: Feature[] = [
     description: 'Alternative USB implementation that might help solve USB issues.',
   },
   {
-    shadow: true,
     type: 'toggle',
+    name: 'LEDGER_COUNTERVALUES_API',
+    valueOn: 'http://countervalue-service.dev.aws.ledger.fr',
+    valueOff: 'https://countervalues.api.live.ledger.com',
+    title: 'Experimental Countervalues API',
+    description: 'Experimental Countervalues API upgrade',
+  },
+  {
+    type: 'integer',
+    name: 'KEYCHAIN_OBSERVABLE_RANGE',
+    title: 'Gap Limit',
+    description: 'Custom gap limit for all accounts',
+    minValue: 20,
+    maxValue: 999,
+  },
```

Comment:
title: 'Custom gap limit',
description: 'Custom gap limit for all accounts. Increasing this value above its default value (20) scans more unused public addresses for coins. Advanced users only, this may break compatibility when restoring your accounts.',